source/common/tracing/zipkin/span_context.h

```diff
@@ -96,11 +70,24 @@ class SpanContext {
    */
   std::string traceIdAsHexString() const { return Hex::uint64ToHex(trace_id_); }
 
+  /**
+   * @return the sampled flag.
+   */
+  bool sampled() const { return sampled_; }
+
+  /**
+   * @return the sampled flag as a 1-character string.
+   */
+  std::string sampledAsString() const {
```

Comment:
Is this used?

Reply:
Will update - the other `...AsString()` aren't used now either, so will remove.